README.md

```diff
@@ -1,7 +1,13 @@
 # Budgy Tracker
 
+| Mobile version |
+|-----------------|
+| ![App](./readme_imgs/mobile_ver.gif) |
+
 > The Budgy Tracker app is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
```

Comment:
Please ensure that you record a video presenting the features of the project and add the video link to your `README file` according to the project requirement.

Reply:
Hi @micheaol, thanks for your kind feedback!, This petition was already implemented in commit 6012dc2. Thanks anyway.